easybuild/easyconfigs/t/Tk/Tk-8.6.4-intel-2015a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.4'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for building
+ a graphical user interface (GUI) in many different programming languages."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%(namelower)s%(version)s-src.tar.gz']
+
+dependencies = [
+    ('Tcl', version),
+    ('libX11', '1.6.3', '-Python-2.7.9'),
```

Comment:
Tcl 8.6.4 clashes with the Tcl version implied by Python (via SQLite), i.e. 8.6.3...